### PR TITLE
Small suggestion to save resources for local dev

### PR DESCRIPTION
**Pull request by salaboy**
Signed-off-by: salaboy <Salaboy@gmail.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

This is just a suggestion. Feel free to disregard it, but I used that flag all the time for dev purposes.

### CODE REVIEW

**Comment by salaboy** [comment]
this is my first PR to the Dapr org, so I might be missing some basic stuff.. please advice.

File: daprdocs/content/en/getting-started/tutorials/configure-state-pubsub.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ helm install redis bitnami/redis --set image.tag=6.2
 ```
 
 For Dapr's Pub/sub functionality, you'll need at least Redis version 5. For state store, you can use a lower version. 
+Notice that by adding `--set architecture=standalone` create a single replica redis setup which can save memory and resources if you are in working in a local environment.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/tutorials/configure-state-pubsub.md]:
```suggestion
Notice that adding `--set architecture=standalone` creates a single replica Redis setup, which can save memory and resources if you are working in a local environment. 
```

**Review by hhunter-ms — APPROVED** [review]
lgtm

File: daprdocs/content/en/getting-started/tutorials/configure-state-pubsub.md [hunk 2]
```diff
@@ -46,6 +46,7 @@ helm install redis bitnami/redis --set image.tag=6.2
 ```
 
 For Dapr's Pub/sub functionality, you'll need at least Redis version 5. For state store, you can use a lower version. 
+Notice that adding `--set architecture=standalone` creates a single replica Redis setup, which can save memory and resources if you are working in a local environment.
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/tutorials/configure-state-pubsub.md]: I just would rephrase this to


```suggestion
Note that adding `--set architecture=standalone` to the `install` command creates a single replica Redis setup, which can save memory and resources if you are working in a local environment. 
```

**Comment by salaboy** [comment]
@yaron2 I don't know what is the `validate-links` issue here.. not sure what to do to unblock this PR
